usecases/MinimumWorkingExample/dodo.py

```diff
@@ -120,4 +123,29 @@ def task_export_knowledgeGraph_emodul():
                 'file_dep': [metadata_file_path, processed_data_file_path],
                 'targets': [knowledge_graph_file],
                 'clean': [clean_targets]
-            }
\ No newline at end of file
+            }
+
+# perform calibration
+def task_perform_calibration():
+    # create folder, if it is not there
+    Path(calibrated_data_directory).mkdir(parents=True, exist_ok=True)
+
+
+
+    # TODO implement "expensive" option on all files!!!
```

Comment:
could we have that already here to see if the general implementation is working (even though the CI uses only the cheap version?)